README.md

```diff
@@ -30,33 +31,112 @@ global:
 Take a look at the [`examples`](https://github.com/webpack/webpack/tree/master/examples) folder.
 
 
-
 # Features
 
 ## Plugins
 
-webpack has a [rich plugin interface](http://webpack.github.io/docs/plugins.html). Most of the features within webpack itself use this plugin interface. This makes webpack very **flexible**.
+Webpack has a [rich plugin
+interface](http://webpack.github.io/docs/plugins.html). Most of the features
+within webpack itself use this plugin interface. This makes webpack very
+**flexible**.
 
 ## Performance
 
-webpack uses async I/O and has multiple caching levels. This makes webpack fast and incredibly **fast** on incremental compilations.
+Webpack uses async I/O and has multiple caching levels. This makes webpack fast
+and incredibly **fast** on incremental compilations.
 
 ## Loaders
```

Comment:
I think we can also provide a link to ["webpack" keyword on npm](https://www.npmjs.com/browse/keyword/webpack) for further results besides highlighted plugins/loaders.